### PR TITLE
SPLAT-870 - bump/tools: update tools image to sync for security and sync with CLI

### PR DESCRIPTION
**Pull request by mtulio**
https://issues.redhat.com/browse/SPLAT-870

Update the Tools image for those reasons:
- update OC client to 4.11
- sync sonobuoy version with CLI: https://github.com/redhat-openshift-ecosystem/provider-certification-tool/blob/main/go.mod#L13
- security updates from Alpine
Screenshot from 2022-11-09 14-01-03

### CODE REVIEW

**Comment by mtulio** [comment]
@bostrt ptal?

File: openshift-tests-provider-cert/hack/build-image.sh [hunk 1]
```diff
@@ -23,15 +23,15 @@ VERSION_PLUGIN_DEVEL="${VERSION_DEVEL:-}";
...
+export VERSION_TOOLS="v0.0.0-alp3156-oc41113-s05610"
+export VERSION_SONOBUOY="v0.56.10"
+export VERSION_OC="4.11.13"
 
 IMAGE_PLUGIN="${REGISTRY_PLUGIN}/openshift-tests-provider-cert"
 IMAGE_TOOLS="${REGISTRY_TOOLS}/tools"
 IMAGE_SONOBUOY="docker.io/sonobuoy/sonobuoy"
 
-export CONTAINER_BASE="alpine:3.14"
+export CONTAINER_BASE="alpine:3.15.6"
```
Comment (bostrt) [inline openshift-tests-provider-cert/hack/build-image.sh:34]: Is this used anywhere?

Comment (bostrt) [reply openshift-tests-provider-cert/hack/build-image.sh:34]: Nevermind :) I see it in the Dockerfile. I haven't seen this done before with env vars. Very nice :)

**Pull request merged**